Gruntfile.js

```diff
@@ -219,7 +219,6 @@ module.exports = function (grunt) {
         src: 'softwerkskammer/testApp',
         options: {
           coverageFolder: 'softwerkskammer/coverageApp',
-          excludes: ['**/activitystore.js'],
```

Comment:
Hängt es vielleicht damit zusammen, dass der code in den anderen Tests (mit DB) schon erfasst wird?